feature-libs/product/multi-dimensional/components/multi-dimensional-container/multi-dimensional-selector/product-multi-dimensional-selector.component.html

```diff
@@ -1,43 +1,72 @@
 <ng-container *ngIf="product">
   <div class="variant-generic-selector">
     <ng-container *ngFor="let category of categories">
-      <label for="{{category.name}}">{{ category.name }}</label>
+      <label>
+        <span>
+          <ng-container
+            *ngIf="
+              getSelectedValue(category.name) as selectedValue;
+              else tmpWithoutLabel
+            "
+          >
+            {{ category.name }}: {{ selectedValue }}
```

Comment:
This looks strange when associated with a select element. Wouldn't it be better if we add this only if there are images? Also, would prefer if it was in the method. 
```typescript
  getCategoryName(category): string {
    if (category.hasImages) {
      const selectedValue = this.getSelectedValue(category.name);
      return `${category.name}${selectedValue ? `: ${selectedValue}` : ''}`;
    }

    return category.name;
  }
```

Reply:
changed